### DISCUSSION TITLE
Uninstall Process

### DISCUSSION THREAD

**Discussion by paolotormon** [zidoro/pomatez#286] [Q&A] [answered]
It does not appear on my Control Panel. Is deleting AppData/Local/Progams/pomatez enough/safe?
![image](https://user-images.githubusercontent.com/61226023/183408973-74f02c84-aedf-41d1-8cb2-0881c5aec7ad.png)

**Comment by sekwah41** [comment]
Thats strange, I believe it was there when I used it on windows though I will need to look into that.

We are currently doing a backend rewrite so I will check that this doesnt happen on the rewrite.

**Comment by gillisandrew** [comment, marked as answer]
Sounds like a bug or an issue with windows. The installer can be run manually from `%LOCALAPPDATA%\Programs\pomatez`.

**Reply by sekwah41** [reply]
Can confirm that it should be there. If its not there, then there is likely something wrong with electron-builder or its been deleted somehow.
